test/specs/cartesian/YAxisSpec.js

```diff
@@ -123,7 +122,7 @@ describe('<YAxis />', () => {
   it('Render 1 ticks when domain={["auto, "auto"]} and dataKey is "noExist" ', () => {
     const wrapper = mount(
       <AreaChart width={600} height={400} data={data}>
-        <YAxis stroke="#ff7300" domain={["auto", "auto"]} />
+        <YAxis stroke="#ff7300" domain={['auto', 'auto']} />
```

Comment:
Are these redundant formatting checks a result of the linting rules setup in the repo? If yes, we should do a single formatting PR to prevent such changes showing up in PRs.

Reply:
Yes i think these were caused by the pre-commit hook on the project. Per @ckifer's comment about upcoming refactors to these tests, i'll revert them and commit without the formatting step